src/types/types.js

```diff
@@ -19,3 +19,8 @@ export type Matrix = {
   e: number,
   f: number
 };
+
+export type SketchbookDocument = {
+  type: "SketchbookDocument",
+  layers: Array<TypeLayer>
```

Comment:
Got a `TypeLayer` here… which should be throwing an exception somewhere. 🤔 I wonder why the CI check didn't pick this up?

Reply:
Ah woops. I didn't realise I had to put `@flow` in the type file. Fixed!